src/cli.rs

```diff
@@ -40,7 +40,10 @@ pub enum SubCommand {
         key: String,
     },
     /// Print all of the secrets
-    Print,
+    Print {
+        /// Secrets output style, possible values are: setenv, json, yaml
```

Comment:
What's the default value here? It would be good to include it in the comment. Alternatively: make this a `style: PrintStyle` and provide an attribute that provides a default, which would work better as an option instead of an argument, e.g.:

```rust
#[clap(long, short, default_value = "setenv")]
style: PrintStyle
```

Reply:
I made this option first, but then realize it may be confusing, since the options are exclusive.

The default value is <empty>, which outputs `KEY=value`.

Reply:
Can you clarify what you mean that the options are exclusive? What would be confusing about it?

Reply:
I think normally, `--arg1` and `--arg2` mean they may be used together. For example, `git log --follow ./ --source`.

Here, at least for now, we do not support printing different formats at the same time. So it cannot be `amber print --json --setenv`. So I made it arguments, and set `print` only takes one argument.

Reply:
I was thinking more along the lines of `--style json`, which is what the `#[clap(...)]` bit I posted above would do.

Reply:
As you wish.